src/app/asset-page/artstor-viewer/artstor-viewer.component.ts

```diff
@@ -406,6 +406,20 @@ export class ArtstorViewerComponent implements OnInit, OnDestroy {
             this.osdViewer.destroy();
         });
 
+        this.osdViewer.addOnceHandler("tile-drawn", () => {
+            // Get all canvas elements, first one is the main canvas, second one is the navigator
+            // The canvas in the reference strip starts as the third one in the collection
+            let canvasElements = document.getElementsByClassName("openseadragon-canvas")
```

Comment:
Wondering if we can use a more specific selector to pick out only the `openseadragon-canvas`s in the `referencestrip`? That way we wouldn't need always start at i = 2

Reply:
Could do something similar to this https://github.com/ithaka/aiw-ui/pull/1207/files#diff-1373abb83f1b3bd7243873880bf670f8R12 ?

Reply:
I use a more specific selector now to get rid of the hardcoded `2`